README.md

```diff
@@ -38,6 +38,11 @@ conda env create -f environment.yml
 source activate devito
 pip install -e .
 ```
+or alternatively install Devito only (without tests and examples) using `pip`:
```

Comment:
I don't wanna claim we support this, though "we're trying our best". Conda should be **the** way. So I'd drop this whole paragraph

Reply:
I'm still against this

Reply:
I don't agree, and I don't think that's what we agreed on Monday.

Reply:
You are assuming that people should only ever use Devito as a standalone. We have a python module that people may want to integrate with different work. 
The same way we don't have to create a *virtualenv* for every requirement we have, people should be able to install devito as a requirement in their own project and therefore Devito has to be pip installable.
We have a pythin package, and no matter how we use it, if we want it to survive in the Python enviromnent it needs to be pip installable like all are requirements are.

Reply:
I think you're not getting my point here. You're absolutely right, that's why I supported this PR since the very beginning, you probably remember that. But here I'm talking about **our** installation instructions, not how some other packages pretend to use or install Devito (that's their job).

Reply:
Fine with me then. This is now tested and requirement automatically reads from the `requirement.txt` so we should be fine as of now. Removed the extra line from the README.